### ISSUE TITLE
Domain Queue: Unable to add domains

### ISSUE THREAD

**Issue by jennec** [domainmod/domainmod#32]
I have added Godaddy Domain Registrar, Registrar Account with API Key and Secret, and no IP.  But when I try to import all Domains Domain Queue just sits there stating "Processing".  It has done for approx 15 hours before I stopped it.

I also tried specifying one domain instead, and the same thing happens.

I turned on debug log, but all I see is cron jobs kicking off.  

Currently I do not seem to be able to add domains from GoDaddy.

This is a new install, so perhaps I have missed a step somewhere,.

Additionally, when I add a domain manually, and add a fee for the TLD, The domain does not show up in the domains list.

**Comment by jennec** [comment]
Apache error logs have the following errors:
[Fri Jun 30 07:22:57.460891 2017] [:error] [pid 2176] [client 10.0.XXX.XXX:51612] PHP Fatal error:  Uncaught Error: Call to undefined function DomainMOD\\curl_init() in /var/www/domainmod/classes/DomainMOD/GoDaddy.php:49\nStack trace:\n#0 /var/www/domainmod/classes/DomainMOD/GoDaddy.php(65): DomainMOD\\GoDaddy->apiCall('API_KEY...', 'API_SECRET...', 'https://api.god...')\n#1 /var/www/domainmod/classes/DomainMOD/DomainQueue.php(111): DomainMOD\\GoDaddy->getDomainList('API_KEY...', 'API_SECRET...')\n#2 /var/www/domainmod/admin/scheduler/run.php(182): DomainMOD\\DomainQueue->processQueueList()\n#3 {main}\n  thrown in /var/www/domainmod/classes/DomainMOD/GoDaddy.php on line 49, referer: http://domains.studygroup.com/admin/scheduler/


Hopefully this will help shed some light on this issue.

**Comment by jennec** [comment]
I am just playing at this, as I don;t know all that much about coding, but I did a search from the root directoriy and got the following results:
grep -r "curl_init"


classes/DomainMOD/DwBuild.php:        $curl = curl_init(); // Create Curl Object
classes/DomainMOD/DnSimple.php:        $handle = curl_init($full_url);
classes/DomainMOD/Conversion.php:        $handle = curl_init($full_url);
classes/DomainMOD/NameCom.php:        $handle = curl_init($full_url);
classes/DomainMOD/Freenom.php:        $handle = curl_init($full_url);
classes/DomainMOD/Goal.php:            $handle = curl_init();
classes/DomainMOD/OpenSrs.php:        $handle = curl_init('https://rr-n1-tor.opensrs.net:55443'); // Production Environment
classes/DomainMOD/OpenSrs.php:        // $handle = curl_init('https://horizon.opensrs.net:55443'); // Test environment
classes/DomainMOD/ResellerClub.php:        $handle = curl_init($full_url);
classes/DomainMOD/Dynadot.php:        $handle = curl_init($full_url);
classes/DomainMOD/GoDaddy.php:        $handle = curl_init($full_url);
classes/DomainMOD/NameSilo.php:        $handle = curl_init($full_url);
classes/DomainMOD/DreamHost.php:        $handle = curl_init($full_url);
classes/DomainMOD/Enom.php:        $handle = curl_init($full_url);
classes/DomainMOD/AboveCom.php:        $handle = curl_init($full_url);
classes/DomainMOD/NameBright.php:            $handle = curl_init($full_url);
classes/DomainMOD/NameBright.php:            $handle = curl_init($full_url);
classes/DomainMOD/Namecheap.php:        $handle = curl_init($full_url);
classes/DomainMOD/InternetBs.php:        $handle = curl_init($full_url);
classes/DomainMOD/Fabulous.php:        $handle = curl_init($full_url);
classes/DomainMOD/System.php:            $handle = curl_init();

I cannot see a function being created anywhere.  Shouldn't there be one, or am I barking up the wrong tree?

**Comment by jennec** [comment]
So I've answered my own issue.  The domain queue process uses a PHP extension called php-curl.  Which is not installed by default on a vanilla php installation.  installing this via apt-get resolved the issue for the domain queue.  

A list of perquisites for during the installation documentation would be helpful to avoid this kind of issue for others.  

But now that I have added my 195 domains to DomainMod successfully, I still can't see them.  Closing this issue and continuing the display of domains problem in Issue #29
